wordpress-easy-301-redirects.php

```diff
@@ -51,6 +51,44 @@ public function getRedirectsJson() : string
      * @return void
      */
     public function redirect() {
+        $userrequest = str_ireplace(get_option('home'),'',$this->getAddress());
+        $userrequest = rtrim($userrequest,'/');
+        $currentState = $this->getRedirects();
+        $redirects =  $currentState['store'];
+        $wildcard = $currentState['wildcard'];
```

Comment:
is there a reason why we shouldn't always have wildcard support? we could then remove all related conditions an branching

Reply:
Is a current feature to allow wildcards or not. In fact. some of our own sites don't have wildcard activated. 

Let me know if you want to go full forced wildcard then

Reply:
Do you see any problems with always allowing wildcards?

Reply:
Besides the fact of removing an user option, the only thing is that if not activated the execution is just one line, but in reallity I don't think that is too much consuming logic in those extra instructions that handle the wildcard:

Reply:
ok yes, but then let's just have wild-card support enabled always, so we don't have to bother with  distinguishing between two states.

Reply:
ok will do

Reply:
Actually the code block is checking that if the redirect is only a wildcard `* ` we dont block the wp admin dashboard, so we are not saving to much code or branching, but I make the wildcard always active regardless

Reply:
applied